notebook-api/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alpine:3.8
+MAINTAINER Hail Team <hail@broadinstitute.org>
+
+RUN apk add \
+  bash \
+  gcc \
+  libffi-dev \
+  musl-dev \
+  openssl-dev \
+  python3 \
+  python3-dev && \
+  pip3 install -U pip && \
```

Comment:
Does this Dockerfile successfully build for you? I've found that pip >= 19 fails on the next line.